PointFree.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,9 +2,18 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:PointFree.xcodeproj">
+      location = "group:README.md">
```

Comment:
Adding the `README` and whatnot to the workspace.

<img width="271" alt="image" src="https://user-images.githubusercontent.com/658/38820277-6fda6e56-416b-11e8-86bd-b7b0e90453d2.png">

Reply:
oh great!